### PR TITLE
Add info

### PR DESCRIPTION
**Pull request by julianoes**
This is a try to add the minimal required info functionality to get the version.

### CODE REVIEW

File: Dronecode-SDK-Swift/Info.swift [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                let version = Version(flightSwMajor: getVersionResponse.version.flightSwMajor,
+                                      flightSwMinor: getVersionResponse.version.flightSwMinor,
+                                      flightSwPatch: getVersionResponse.version.flightSwVendorPatch,
+                                      flightSwVendorMajor: getVersionResponse.version.flightSwVendorMajor,
+                                      flightSwVendorMinor: getVersionResponse.version.flightSwVendorMinor,
+                                      flightSwVendorPatch: getVersionResponse.version.flightSwVendorPatch,
+                                      osSwMajor: getVersionResponse.version.osSwMajor,
+                                      osSwMinor: getVersionResponse.version.osSwMinor,
+                                      osSwPatch: getVersionResponse.version.osSwPatch)
+                // FIXME: remove debug prints
```
Comment (JonasVautherin) [inline Dronecode-SDK-Swift/Info.swift]: I guess that should go away before it gets merged :sunglasses:

Comment (julianoes) [reply Dronecode-SDK-Swift/Info.swift]: Yes

**Comment by julianoes** [comment]
I'll test it with Yuneec SITL where we have the vendor version populated.

File: Dronecode-SDK-Swift/Info.swift [hunk 2]
```diff
@@ -0,0 +1,95 @@
+//
```
Comment (JonasVautherin) [inline Dronecode-SDK-Swift/Info.swift]: Can we remove the copyright? :sweat_smile:

Comment (julianoes) [reply Dronecode-SDK-Swift/Info.swift]: done.

**Pull request merged**